app/src/main/java/org/mozilla/focus/utils/FirebaseHelper.java

```diff
@@ -236,6 +237,13 @@ protected void onPostExecute(Void result) {
         return remoteConfigDefault;
     }
 
+    @Override
+    void refreshRemoteConfigDefault(Context context) {
+        remoteConfigDefault = null;
+        getRemoteConfigDefault(context);
```

Comment:
Maybe we don't need this line getRemoteConfigDefault(context);
It's quite unlikely but if during this getRemoteConfigDefault() another getRemoteConfigDefault() comes in things become nasty.
If you agree, I might also rename this function to clearRemoteConfigDefault